webpack/assets/javascripts/react_app/components/HostDetails/Status/StatusIcon.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  ExclamationTriangleIcon,
+  CheckCircleIcon,
+  ExclamationCircleIcon,
+  BanIcon,
+} from '@patternfly/react-icons';
+import Skeleton from 'react-loading-skeleton';
```

Comment:
do we have this dependency in `foreman-js`?

Reply:
[yes](https://github.com/theforeman/foreman-js/blob/69932d323fea6f8ac29febd653366c4bde7804e2/packages/vendor-core/package.json#L74)